test/Icicle/Test/Arbitrary.hs

```diff
@@ -24,7 +26,7 @@ instance Arbitrary Attribute where
 
 instance Arbitrary DateTime where
   arbitrary =
-    DateTime <$> elements muppets -- FIX replace with an actual DateTime
+    dateOfYMD <$> oneof_vals [2010..2014] <*> oneof_vals [1..12] <*> oneof_vals [1..28]
```

Comment:
If you have a look at DateTimeUtils in ivory you'll find a days number to date function which would be better than this (and cover all edges, leap years...).